docs/taints-tolerations.md

```diff
@@ -0,0 +1,92 @@
+# Taints and Tolerations Workload
+
+The Taints and Tolerations workload playbook is `workloads/taints-tolerations.yml` and will run the Taints and Tolerations  workload on your cluster.
+
+The Taints and Tolerations workload's purpose is to validate if the OpenShift cluster can deploy 130 hello-pods with memory and CPU requests per tainted worker node.
```

Comment:
Should we make the number of pods, tainted nodes and cpu requests configurable so that it will work/scale for any sized cluster?

Reply:
This will be moved as an enhancement/issue for addressing later.

Reply:
Sounds good.